doc: Add notes for configuring TLS in multi-tenant TTS env (#580)
TheThingsIndustries/lorawan-stack-docs: doc/content/getting-started/installation/certificates.md

    └── ttn-lw-stack-docker.yml    # configuration file for {{% tts %}}
```

> {{< note >}} If you are using Let's Encrypt in a multi-tenant {{% tts %}} environment, make sure you specify all tenant addresses in TLS configuration. [Read more]({{< ref "/getting-started/installation/configuration#tls" >}}) {{</ note >}}

Reviewer: ```suggestion
{{< note >}} If you are using Let's Encrypt in a multi-tenant {{% tts %}} environment, make sure you [specify all tenant addresses]({{< ref "/getting-started/installation/configuration#tls" >}}) in the TLS configuration of `ttn-lw-stack-docker.yml`.{{</ note >}}
```
